debian/cocaine-v12-runtime.init

```diff
@@ -14,7 +14,7 @@
 # Author: Andrey Sibiryov <kobolog@yandex-team.ru>
 
 PATH=/sbin:/bin:/usr/sbin:/usr/bin
-NAME=cocaine-runtime
+NAME=cocaine-v12-runtime
```

Comment:
If it's a Yandex-related problem, solve it internally, don't put this bullshit here.